utils/useful.py

```diff
@@ -323,14 +324,19 @@ def empty_page_format(_: Any, __: Any, entry: T) -> T:
     return entry
 
 
+P = ParamSpec("P")
+
+
 class ListCall(List[Any]):
     """Quick data structure for calling every element in the array regardless of awaitable or not"""
-    def append(self, rhs: Awaitable[Any]) -> None:
-        return super().append(rhs)
+    def add(self, rhs: Callable[P, Awaitable[T]]) -> Callable[P, Awaitable[T]]:
```

Comment:
???

Reply:
i just want it to feel like a list, idk why you want this to be similar to a set